wikimedia/wikipedia-ios
# [ALT TEXT] Bottom sheet - part 1

**Phabricator:** https://phabricator.wikimedia.org/T370232

### Notes
* This creates a barebones version of the bottom sheet
* It doesn't match the designs yet
* I'm still waiting on design's update on the icon we should use [see ticket](https://phabricator.wikimedia.org/T370232#10028171)
* Using a text view for multiline purposes created the need to add the placeholder as a label
* I'm updating ArticleViewController init method to receive the alt text view model, let me know if this is not desired (I can create a separate init, adding the VM after creating the VC was causing to be nil when needed)
* The VM property, when kept weak, did not hold the value correctly. It was left strong and nulled on the existing deinit method. 

### Test Steps
1. Go through all the steps to get into alt text - tweaking some values might be necessary
2. When prompted to add alt text, tap "add"
3. Verify the basics of the design, dark mode, dynamic type
4. The placeholder should disappear when typing
5. The button should enable only with text = there was no character minimum defined
